database/data.js

```diff
@@ -0,0 +1,87 @@
+var faker = require('faker');
+var csv = require('csv-parser');
+var fs = require('fs');
+
+//set seed
+faker.seed(123);
+
+var generateRating = function(max){
+  // return random Number between 0 and max
+  var randomNum = parseInt(Math.random() * max)
+  return randomNum;
+};
+
+
+// generate data
+var data_bookInfo = function(){
+  var bookInfo = {
+    title : faker.random.words(3),
+    author : faker.name.findName(),
+    description : faker.lorem.sentences(6),
+    num_reviews : faker.random.number({"min" :1, "max" : 50000}),
+    num_ratings : faker.random.number({"min" :1, "max" : 2000000}),
+    ratings_star : faker.random.number(0, 5, 2),
+    image :faker.image.image()
```

Comment:
I see you have a space before your colons. What linter are you using? The airbnb one gives me grief if I do this.

Reply:
I'm curious about the space too. I thought it would not work.

Reply:
Forgot to set up lint for this. Checking  through and fixing all issues